src/components/BackButton.tsx

```diff
@@ -19,14 +19,15 @@ const styles = StyleSheet.create( {
   },
 } )
 
-const BackButton = ( { label = 'Cancel', ...props }: BackButtonProps ) => {
+const BackButton = ( { label: Label = 'Cancel', ...props }: BackButtonProps ) => {
   const navigation = useNavigation()
 
   const goBack = () => navigation.goBack()
 
   return (
     <TouchableOpacity onPress={goBack} {...props}>
-      <Text style={styles.label}>{label}</Text>
+      {Label && <Text style={styles.label}>{Label}</Text>}
+      {!Label && <Label />}
```

Comment:
Is this impossible lol?

Label => falsy, followed by a render of a falsy value

Looks to me like `label` should just be named `label`

Reply:
Well I was trying to do if `Label` is a string then use the render the string inside the `<Text>`. If it is a JSX element it should render that and that is why I had to rename `label`->`Label` since JSX components need to be `CamelCase`

Reply:
I see what you meant. Fixed it in 73677a6